packages/jest-koa-mocks/src/create-mock-context/create-mock-context.ts

```diff
@@ -70,17 +69,14 @@ export default function createContext<
     state,
   };
 
-  const protocolFallback = encrypted ? 'https' : 'http';
-  const urlObject = new URL(url, `${protocolFallback}://${host}`);
-
   const req = httpMocks.createRequest({
-    url: urlObject.toString(),
+    url,
```

Comment:
now it is necessary to process fully-qualified url on one's own before passing it to `createMockContext`, something what that function did internally and worked fine for both relative and absolute URLs before this PR :)
```javascript
const { host, protocol, pathname, searchParams } = new URL(url);

const ctx = createMockContext({
  url: pathname + searchParams,
  host,
  encrypted: protocol === 'https',
});
```

but I see the point, maybe that url: urlObject.toString() was wrong on line 77, since you expected only the relative part of the URL to be there. So, this could be used instead to ensure that:

```suggestion
    url: urlObject.pathname + urlObject.searchParams,
```

cheers 🍻